packages/next/src/build/babel/loader/get-config.ts

```diff
@@ -273,6 +273,13 @@ function getFreshConfig(
       return false
     }
 
+    if (
+      filename.includes('/node_modules/') ||
```

Comment:
Will use [\\/]node_modules[\\/] faster than 2 includes check for corner cases? Also easier to write